internal/pkg/deploy/job.go

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package deploy holds the structures to deploy infrastructure resources.
+// This file defines service deployment resources.
+package deploy
+
+// DeleteJobInput holds the fields required to delete a service.
+type DeleteJobInput struct {
```

Comment:
Does this have the same field names as `DeleteServiceInput` in `service.go`? If so, I wonder if we can just rename the struct to DeleteWorkloadInput. Where does this get called? Execute?

Reply:
Yes, in Execute. I had made this change in a subsequent branch, but have now made it here as well. Thanks!